pokemon_battle_arena/src/db/pokemon_token.rs

```diff
@@ -170,15 +165,13 @@ impl PokemonToken {
             self.move_four = Some((moves[3].clone(), moves[3].get_power_points().unwrap()));
         }
     }
-
-    pub fn set_type(&mut self, position: u8, change: enums::types) {
+    pub fn set_type(&mut self, position: u8, change: enums::Types) {
         match position {
             0 => self.type_one = change,
             1 => self.type_two = change,
             _ => unreachable!(),
         }
     }
```

Comment:
This again, is not an idiomatic Rust API. The method is not even documented: why would the caller know that only "0" and "1" are allowed?